Generate_tile.scad

```diff
@@ -38,27 +41,27 @@ module letter(l) {
 }
 
 
-text_height = (tile_length - letter_size);
-text_centre = (tile_width / 2);
+text_height = (final_length - letter_size);
+text_centre = (final_width / 2);
 
-translate([text_height,text_centre,0]) rotate([0,0,90]) letter(text);
+translate([text_height,text_centre,tile_height]) rotate([0,0,90]) letter(text);
 
 
 
 // Make the Tile
 
-margin_length_offset = (tile_length - (margin * 2));
-margin_width_offset = (tile_width - (margin * 2));
+margin_length_offset = (final_length - (margin * 2));
+margin_width_offset = (final_width - (margin * 2));
 margin_height_offset = (margin_height / 2);
-mag_x_pos = (tile_width / 2);
-mag_y_pos = (tile_length /2);
+mag_x_pos = (final_width / 2);
+mag_y_pos = (final_length /2);
 mag_radius = (mag_diameter /2);
 
 difference () {
     minkowski() {
-        cube([tile_length,tile_width,tile_height]);
+        cube([final_length,final_width,tile_height]);
         // rounded corners
-        cylinder(r=rounding_factor,h=rounding_factor);
```

Comment:
The rounding_factor was affecting the overall tile height because it was used as height